Tests/BackendIntegrationTests/OtherIntegrationTests.swift

```diff
@@ -173,7 +173,7 @@ class OtherIntegrationTests: BaseBackendIntegrationTests {
         try AvailabilityChecks.iOS15APIAvailableOrSkipTest()
 
         let result = try await self.purchases.productEntitlementMapping().entitlementsByProduct
-        expect(result).to(haveCount(17))
+        expect(result).to(haveCount(18))
```

Comment:
Bringing in the change from https://github.com/RevenueCat/purchases-ios/commit/7f6baa9aa4edd5504e600c155bbdc5315c4336bd to make CI pass